python/grass/jupyter/interact_display.py

```diff
@@ -0,0 +1,228 @@
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains functions for interactive display
+#            in Jupyter Notebooks.
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#            This program is free software under the GNU Gernal Public
+#            License (>=v2). Read teh file COPYING that comes with GRASS
+#            for details.
+
+import os
+import sys
+import subprocess
+import folium
+import grass.script as gs
+import shutil
+import codecs
+
+def create_location(
```

Comment:
What's wrong with `grass.script.create_location`?

Reply:
`create_location` calls `create_environment`. The user cannot pass an environment to `create_location` so I couldn't get a new location to be created in `self._env`; it will always create a new environment with `os.environ.copy()` and doesn't return the new environment either. I think this means it wouldn't work? Or is there another workflow you had in mind?

Reply:
I switched back to `gs.create_location` and it doesn't return an error anymore and it seems to work fine. I'm not sure why I was having so much trouble with that before... So, all set now - thanks!